config.example.toml

```diff
@@ -0,0 +1,144 @@
+# This is an example configuration with all supported properties explicitly set
+# Most properties can be overriden with environment variables specified in this file
+# Most properties also have defaults (mentioned in this file) if they are not set in either the config file or the corresponding environment variable
+
+# GoBinary returns the path to the go binary to use. This value can be a name of a binary in your PATH, or the full path
+# Defaults to "go"
+# Env override: GO_BINARY_PATH
+GoBinary = "go"
```

Comment:
Should it default to go1.11rc?

Reply:
It defaults to `go` because we assume the installation environment has an appropriate version installed.

When not running in a Docker or similar environment where we have an appropriate version of go installed, the operator is expected to set `GO_BINARY_PATH`. Which would currently be `go1.11rc1`, or the full path if the binary is not in the user's `$PATH`.

Reply:
Makes sense!